docs/data/base/components/select/select.md

```diff
@@ -0,0 +1,91 @@
+---
+product: base
+title: React Select unstyled component and hook
+components: SelectUnstyled, MultiSelectUnstyled, OptionUnstyled, OptionGroupUnstyled
+githubLabel: 'component: select'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#combobox
+---
+
+# Select
+
+<p class="description">Select components are used for choosing options from a list.</p>
```

Comment:
Let's make this more informative, as with the `Modal` page. I'm drawing from [the MDN entry for `<select>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) here:

```suggestion
<p class="description">The `SelectUnstyled` and `MultiSelectUnstyled` components let you create menus of options for users to choose from.</p>
```